### PR TITLE
feat: cosmos integration for bounties/hackathons

### PR DESCRIPTION
**Pull request by chibie**
##### Description

Backend changes

- [x] Model & view changes
- [x] Functions to verify transaction from blockchain

Frontend changes

- [x] Funder can create bounty
- [x] Funder can create bounty in hackathon
- [x] Update bounty details page with explorer address/tx links
- [x] Write cosmos_extension.js

##### Testing

bounty payout with keplr wallet -> https://share.vidyard.com/watch/qdisSUytrLufPtVDQ1xuLZ?

### CODE REVIEW

**Comment by chibie** [comment]
pending work on this PR involves fixing `cosmos_extension.js` which involves processing a payout using keplr wallet browser extension to sign and broadcast the transaction

this doc explains how to sign a transaction using keplr + cosmjs -> https://docs.keplr.app/api/cosmjs.html

however, we have to find a way to access `SigningStargateClient` object (`SigningCosmosClient` is for cosmjs/launchpad i.e version 3) by having a browser script of [cosmjs/stargate](https://github.com/cosmos/cosmjs/tree/main/packages/stargate) library to be able to sign the transaction with keplr wallet.

here's what i've written for cosmos_extension.js with relevant comments -> https://github.com/gitcoinco/web/pull/9588/commits/1bd7765a5f4df3717bf18bc79a70deaeffce0860

**Review by thelostone-mc — APPROVED** [review]

File: app/assets/v2/js/pages/bounty_detail/cosmos_extension.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+  }
+
+  try {
+    await window.keplr.enable(chainId);
+  } catch (e) {
+    _alert({ message: gettext('Please unlock Keplr browser wallet extension.') }, 'danger');
+    return;
+  }
+
+  let client = await window.CosmWasmJS.setupWebKeplr({
```
Comment (schultztimothy) [inline app/assets/v2/js/pages/bounty_detail/cosmos_extension.js:18]: Looking great @chibie ! Do you by chance have a link to the JS library/documentation. I can't seem to track it down

Comment (chibie) [reply app/assets/v2/js/pages/bounty_detail/cosmos_extension.js:18]: https://cosmwasm.github.io/CosmWasmJS/

File: app/dashboard/views.py [hunk 2]
```diff
@@ -6520,7 +6562,24 @@ def payout_bounty_v1(request, fulfillment_id):
         fulfillment.save()
         record_bounty_activity(bounty, user, 'worker_paid', None, fulfillment)
 
-    elif payout_type in ['qr', 'web3_modal', 'polkadot_ext', 'harmony_ext', 'binance_ext', 'rsk_ext', 'xinfin_ext', 'nervos_ext', 'algorand_ext', 'sia_ext', 'tezos_ext', 'casper_ext']:
+    elif (
+        payout_type
+        in [
+            'qr',
```
[the remark this answers is not in the document]
Comment (chibie) [reply app/dashboard/views.py:6568]: `web3_modal` and `fiat` aren't in 6339

**Pull request merged**